searchconsole/auth.py

```diff
@@ -20,7 +20,7 @@
 from .account import Account
 
 
-def authenticate(client_config, credentials=None, serialize=None):
+def authenticate(client_config, credentials=None, serialize=None, from_colab=False):
```

Comment:
I think we can be more generic than referring to Google Colab. What if we had a parameter like `flow` that could be `web` by default and optionally `console` for environments (like Colab) where the web flow isn't possible?

Reply:
@joshcarty sounds good!

Reply:
> Guys, any chance to get this merged?
> 
> Thanks

@joshcarty I created a new, clean and tested pull request per your comments here https://github.com/joshcarty/google-searchconsole/pull/24

Please feel free to close this one. It was not immediately obvious to me how to make the changes to the existing pull request.

Cheers